.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: groovy
```

Comment:
See your previous Github Actions config file for CI:
https://github.com/daviesdrew/variantcalling/blob/e823d4c1e6b3dc44149cca981a60c5e4497d4eb5/.github/workflows/ci.yml#L1-L30

Reply:
.travis.yml has been edited to be more precise using some of the formatting from .github actions but since I am using .travis.yml it seems the above config didnt work but I will continue improving travis.yml